webvh/webvh/did/utils.py

```diff
@@ -1,42 +1,42 @@
 """Utilities for shared functions."""
 
-from acapy_agent.wallet.error import WalletDuplicateError
-from acapy_agent.wallet.keys.manager import (
-    MultikeyManager,
-    MultikeyManagerError,
-)
-from acapy_agent.vc.data_integrity.manager import DataIntegrityManager
-from acapy_agent.vc.data_integrity.models.options import DataIntegrityProofOptions
-
-WITNESS_CONNECTION_ALIAS_SUFFIX = '@witness'
-
-KEY_ID_SUFFIXES = {
-    'next': '#next',
-    'update': '#update',
-    'witness': '#witness'
-}
+import base64
+import jcs
+import json
+import hashlib
+from datetime import datetime, timezone
+from multiformats import multibase, multihash
+
+from aiohttp import ClientResponseError, ClientSession
+
+from did_webvh.core.state import DocumentState
+
+WITNESS_CONNECTION_ALIAS_SUFFIX = "@witness"
 ALIAS_PURPOSES = {
-        'witnessConnection': '@witness',
-        'nextKey': '@nextKey',
-        'updateKey': '@updateKey',
-        'witnessKey': '@witnessKey',
-    }
+    "witnessConnection": "@witness",
+    "nextKey": "@nextKey",
+    "updateKey": "@updateKey",
+    "witnessKey": "@witnessKey",
+}
 
-def create_alias(identifier: str, purpose: str):
-    """Get static alias."""
-    return f'webvh:{identifier}{ALIAS_PURPOSES[purpose]}'
+
+def version_time():
+    """Create versionTime timestamp for log entries."""
+    return str(datetime.now(timezone.utc).isoformat("T", "seconds")).replace(
+        "+00:00", "Z"
+    )
 
 
-def key_to_did_key_vm(multikey: str):
-    """Transform a multikey to a did key verification method."""
-    return f"did:key:{multikey}#{multikey}"
+def create_alias(identifier: str, purpose: str):
+    """Get static alias."""
+    return f"webvh:{identifier}{ALIAS_PURPOSES[purpose]}"
 
 
 def server_url_to_domain(server_url: str):
     """Replace %3A with : if domain is URL encoded."""
```

Comment:
Any reason to just handle this url-encoded character, rather than the whole set (i.e.: returning the full url-decoded url)?

Reply:
I think the goal is only to cater to the port delimiter being encoded rather than all characters